libs/js-sdk/src/tests/http-client.spec.ts

```diff
@@ -334,20 +334,129 @@ describe('http client', () => {
         server.listen()
 
         try {
+            expect.assertions(5)
+
             const httpClient = createBrowserClient({
                 environmentApiKey: 'env-api-key',
                 audiences: [],
                 api: featureBoardHostedService,
                 updateStrategy: { kind: 'manual' },
             })
 
-            await httpClient.updateAudiences(['test-audience'])
+            await httpClient.waitForInitialised()
+
+            expect(httpClient.initialised).toBeTruthy()
+
+            httpClient.subscribeToInitialisedChanged((init: boolean) => {
```

Comment:
Shouldn't need `: boolean` here, it should be inferred from the function you are calling